### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by yanyshyn**
Did all  tasks that I could.

### CODE REVIEW

File: lesson_2/HomeWork_6.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+""""Нарисовать равнобедренный треугольник из символов ^. Высоту выбирает пользователь. """
+if __name__ == '__main__':
+    num = abs(int(input("Enter a number: ")))
+r = 0
+for i in range(1,num + 1):
+    num -= 1
+    print(num*" ",u'\u22C0'*i + u'\u22C0'*r)
+    r += 1
```
Comment (itea-artem) [inline lesson_2/HomeWork_6.py]: New line

File: lesson_2/HomeWork_6.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+""""Нарисовать равнобедренный треугольник из символов ^. Высоту выбирает пользователь. """
+if __name__ == '__main__':
+    num = abs(int(input("Enter a number: ")))
+r = 0
+for i in range(1,num + 1):
```
Comment (itea-artem) [inline lesson_2/HomeWork_6.py]: PEP8: whitespace after comma

File: lesson_2/HomeWork_7.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+"""Напишите программу “угадай число”.  Компьютер генерирует рандомное число (функция random), /
+пользователь вводит числа, пока не угадает, а компьютер отвечает больше или меньше
+"""
+import random
+if __name__ == '__main__':
+    gennumber = random.randrange(1,101,1)
```
Comment (itea-artem) [inline lesson_2/HomeWork_7.py]:
```suggestion
    gen_number = random.randrange(1,101,1)
```

File: lesson_2/HomeWork_7.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    number = abs(int(input("Pick up a number between 1 - 100 : ")))
+while number != gennumber:
+    if gennumber > number:
+        number = abs(int(input( "The number you've chosen less than winning number let's try again: ")))
+    elif gennumber < number:
+        number = abs(int(input("The number you've chosen greater than winning number let's try again: ")))
+    else:
+        break
+print("You WON!!! The winning  number is: ", gennumber)
+
```
Comment (itea-artem) [inline lesson_2/HomeWork_7.py]: Too many blank lines

File: lesson_2/HomeWork_8.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+"""Написать функцию, которая вычисляет среднее арифметическое \
+элементов массива(списка), переданного ей в качестве аргумента.
+"""
+
+
+def arithm_mean(mylist):
```
Comment (itea-artem) [inline lesson_2/HomeWork_8.py]: Function name.

File: lesson_2/HomeWork_8.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    mylist = list(mylist)
```
Comment (itea-artem) [inline lesson_2/HomeWork_8.py]: Change everywhere
```suggestion
    my_list = list(mylist)
```

File: lesson_2/Tack4.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+if __name__ == '__main__':
+    string = input("Enter any String which contain a capital letters: ")
+    capitals = list()
+    for x in range(len(string)):
+        if string[x].isupper():
+            capitals.append(string[x])
+            continue
```
Comment (itea-artem) [inline lesson_2/Tack4.py]: unnecessary continue

File: lesson_2/Tack4.py [hunk 8]
```diff
@@ -0,0 +1,8 @@
+if __name__ == '__main__':
+    string = input("Enter any String which contain a capital letters: ")
+    capitals = list()
+    for x in range(len(string)):
```
Comment (itea-artem) [inline lesson_2/Tack4.py]: c++ for cycle.

File: lesson_2/HomeWork_6.py (hunk 1 above)
Comment (yanyshyn) [reply lesson_2/HomeWork_6.py]: fixed

File: lesson_2/HomeWork_6.py (hunk 2 above)
Comment (yanyshyn) [reply lesson_2/HomeWork_6.py]: fixed

File: lesson_2/HomeWork_7.py (hunk 4 above)
Comment (yanyshyn) [reply lesson_2/HomeWork_7.py]: fixed

File: lesson_2/HomeWork_8.py (hunk 5 above)
Comment (yanyshyn) [reply lesson_2/HomeWork_8.py]: fixed

File: lesson_2/Tack4.py (hunk 7 above)
Comment (yanyshyn) [reply lesson_2/Tack4.py]: fixed

File: lesson_2/HomeWork_8.py (hunk 6 above)
Comment (yanyshyn) [reply lesson_2/HomeWork_8.py]: fixed

File: lesson_2/Tack4.py (hunk 8 above)
Comment (yanyshyn) [reply lesson_2/Tack4.py]: fixed

**Pull request closed without merging**